src/components/ServiceCard.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  Card,
+  CardContent,
+  CardFooter
+} from "@baltimorecounty/dotgov-components";
+
+let starImg = {
```

Comment:
we don't want to use float here, this could cause some layout issues.

we want to:

- set the position of the container to realive
- set the most popular service icon to absolute position and use right and top properties to position.

We probably want to do this through css, but i'm not sure the best way to do that right now, so it can stay as styles.

Reply:
pushed up some style changes to handle this.